shared/validation/validator.py

```diff
@@ -51,11 +51,8 @@ def _normalize_coerce_bundle_analysis_threshold(self, value):
         coerced_value: BundleThreshold = BundleSizeThresholdSchemaField().validate(
             value
         )
-        # This change is not forwards compatible, so we need this step in the rollout
-        from shared.rollouts.features import BUNDLE_THRESHOLD_FLAG
-
-        if BUNDLE_THRESHOLD_FLAG.check_value(identifier=1, default=False):
```

Comment:
I think the `Feature` class has a `check_value_async` https://github.com/codecov/shared/blob/e20db3999ad82d123fa53a0aae01ba2177c94d81/shared/rollouts/__init__.py#L162 . Does that not work here?

Reply:
The function is not async... I'm not sure

Reply:
> The function is not async.

True. I'm not super familiar with how the worker's async context works, so I just wanted to see if that method in `Feature` would be a quick fix. If it's more complicated, then let's defer the investigation to the ticket.